clingwrapper/src/clingwrapper.cxx

```diff
@@ -593,6 +593,16 @@ bool Cppyy::AppendTypesSlow(const std::string& name,
       if (is_integral(i))
         integral_value = strdup(i.c_str());
       types.emplace_back(type, integral_value);
+    } else if (parent && (Cpp::IsNamespace(parent) || Cpp::IsClass(parent))) {
+        if (Cppyy::TCppType_t type = Cppyy::GetTypeFromScope(Cppyy::GetNamed(name, parent))) {
+            const char* integral_value = nullptr;
```

Comment:
We did not address the strdup code duplication

Reply:
Yeah, forgot about it.
There wasn't any review comment.
Will fix it.